sources/packages/web/src/components/generic/StatusInfoLabel.vue

```diff
@@ -0,0 +1,34 @@
+<template>
+  <v-icon :icon="icon" :color="data.status" class="status-info-icon" />
+  <span class="mx-2">
+    <slot>{{ data.header }}</slot>
+  </span>
+</template>
+
+<script lang="ts">
+import { PropType, computed } from "vue";
+import { StatusInfo, StatusInfoDetails } from "@/types";
+
+export default {
+  props: {
+    data: {
+      type: Object as PropType<StatusInfoDetails>,
```

Comment:
For small component like that also are not supposed to be extended much I would prefer two properties instead of a model.